etc/scripts/run-pipeline-scenarios.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+BASE_DIR=$(pwd)
+WORK_DIR=$(pwd)/build
+
+RUNTIME_CONFIG=$(elyra-metadata list runtimes --json | jq -r '[ .[] | {name: .name, schema_name: .schema_name}] |  .[] | select(.schema_name=="kfp") | .name')
```

Comment:
This produces a list when there are multiple kfp runtimes configured.  Does this need something like `| head -1` so that only one is captured?